### PR TITLE
GPII-1759:  Updated to latest `gpii-express` and `infusion`...

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [GPII-1759](https://issues.gpii.net/browse/GPII-1759) for details.

### CODE REVIEW

**Comment by the-t-in-rtf** [comment]
@amb26, hold off on approving this one until we finish with the `gpii-express` pull.

File: package.json [hunk 1]
```diff
@@ -13,13 +13,13 @@
     "url": "https://github.com/GPII/gpii-mail-test"
   },
   "dependencies": {
-    "infusion": "2.0.0-dev.20151211T025229Z.eabb4fc",
+    "infusion": "git://github.com/amb26/infusion.git#81ef49cdc0e78a96549c5195d9ffd42604087096",
```
Comment (amb26) [inline package.json]: I've just rolled 2.0.0-dev.20160505T142427Z.63b8a53 which contains all my latest fixes - see if this works for you. To speed the process a bit, you can self-push an update to gpii-express if it just consists of an update to this version and all the tests pass.

Comment (the-t-in-rtf) [inline package.json]: Thanks.  I included the update in an unrelated PR for the package, it works fine.  Will push it through along with the versioned reference to gpii-express.

**Comment by the-t-in-rtf** [comment]
This is ready to review, and fairly self-contained.

**Comment by amb26** [comment]
Merged in - was there a particular reason there are 100 repetitions of the tests?

**Comment by the-t-in-rtf** [comment]
There are 100 iterations because I wanted to ensure that the mail test server can be run repeatedly in dozens of tests without hitting some of the test cleanup problems and port conflicts we have seen in other packages.

**Pull request merged**